packages/management/addons-manager-v2/bundle/config/upstream/addons-manager.yaml

```diff
@@ -240,6 +240,8 @@ spec:
       labels:
         app: tanzu-addons-manager
     spec:
+      #@ if/end data.values.tanzuAddonsManager.deployment.nodeSelectors:
+      nodeSelector: #@ data.values.tanzuAddonsManager.deployment.nodeSelectors
```

Comment:
In TCE PR the templating for nodeSelector is different than here. https://github.com/vmware-tanzu/community-edition/pull/3602/files#diff-30c040f98b052b9387f64035c4e03288fc8ff3e7c5f89a7e7847769101c83be5R25

Which one is correct ?

Reply:
This one is using basefile as template directly. TCE is overlaying on top of a basefile. So they are different.